doc/source/whatsnew/v0.25.3.rst

```diff
@@ -10,6 +10,7 @@ including other versions of pandas.
 
 Bug fixes
 ~~~~~~~~~
+- Bug in missing fill_na parameter to DataFrame.unstack() with list of levels (:issue:`30740`)
```

Comment:
Thanks for adding this. can you move it to doc/source/whatsnew/v1.0.0.rst

Reply:
ok

Reply:
Done